### ISSUE TITLE
Migration ignoring table prefix

### ISSUE THREAD

**Issue by K4q4m3** [spatie/laravel-binary-uuid#31]
After installing this package, and runing php artisan migration
```
 SQLSTATE[HY000]: General error: 1005 Can't create table 'boljacij_new.#sql-357d_1c82b7' (errno: 150) (SQL: alter table `promotion_pages` add constraint `promotion_pages_image_id_foreign` foreign key (`image_id`) references `images`
  (`id`) on delete restrict on update restrict)
```

config/database.php:
```
'connections' => [

        'mysql' => [
            'driver' => 'mysql',
            'host' => env('DB_HOST', 'localhost'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_DATABASE', 'forge'),
            'username' => env('DB_USERNAME', 'forge'),
            'password' => env('DB_PASSWORD', ''),
            'charset' => 'utf8',
            'collation' => 'utf8_unicode_ci',
            'prefix' => 'bc_',
            'strict' => false,
            'engine' => 'InnoDB',
        ],
```
db migration file
```
Schema::create('promotion_pages', function (Blueprint $table) {
            $table->uuid('uuid');
            $table->string('title');
            $table->text('content');
            $table->integer('image_id')->unsigned()->default(1);
            $table->foreign('image_id')->references('id')->on('images')
                ->onUpdate('restrict')->onDelete('restrict');
            $table->primary('uuid');
        });
```

When I remove this package, everyting runs normal.

**Comment by brendt** [comment]
What points to an issue with table prefixes? It looks to me that the migrations fail to run because there's data in the tables you're trying to alter, which make a foreign key constraint fail.

What happens when you run the following **on a test database** `php artisan migrate:fresh`? Does it still fail?

**Comment by K4q4m3** [comment]
Migration creates table without prefix. **promotion_pages** should be **bc_promotion_pages**

In SQLSTATE error you can see attempt of creating foreign key to table **images**, instead of **bc_images**.

**Comment by brendt** [comment]
How is the `id` field in the `images` blueprint built?

**Comment by K4q4m3** [comment]
```
Schema::create('images', function (Blueprint $table) {
            $table->increments('id');
            $table->string('title');
            $table->string('path');
            $table->string('slug')->unique();
            $table->integer('author_id')->unsigned();
            $table->foreign('author_id')->references('id')->on('users')
                ->onUpdate('restrict')->onDelete('restrict');
            $table->timestamps();
            $table->softDeletes();
            
        });
```

**Comment by brendt** [comment]
This issue is fixed in [1.1.2](https://github.com/spatie/laravel-binary-uuid/releases/tag/1.1.2).
